### PR TITLE
PD-560 Use active currency

### PR DESCRIPTION
**Pull request by edwardupton**

### CODE REVIEW

File: src/gaTracker/helpers.ts [hunk 1]
```diff
@@ -310,7 +310,7 @@ export const hasGA3 = (): boolean => !!getWebPropertyId();
 function convertProductsToGa4Format(products: Detail[], sendIndex: boolean): GA4Product[] {
 	return products.map((product) => {
 		const converted = {
-			currency: (LittledataLayer.ecommerce && LittledataLayer.ecommerce.currencyCode) || '',
+			currency: window.Shopify?.currency?.active || LittledataLayer.ecommerce?.currencyCode, //fallback to shop currency
```
Comment (lyykfi) [inline src/gaTracker/helpers.ts:313]: i think we should use getActiveCurrency  here.

**Review by lyykfi — CHANGES_REQUESTED** [review]

File: src/segmentTracker/helpers.ts [hunk 2]
```diff
@@ -86,7 +86,6 @@ export const trackEvents = (): void => {
 				'Product Clicked',
 				{
 					...segmentProduct(product),
-					currency: LittledataLayer.ecommerce.currencyCode,
```
Comment (lyykfi) [inline src/segmentTracker/helpers.ts:89]: hm.... i'm not sure. should we also use getActiveCurrency(), here?

Comment (lyykfi) [reply src/segmentTracker/helpers.ts:89]: @edwardupton

Comment (edwardupton) [reply src/segmentTracker/helpers.ts:89]: I removed it because we already fetch the currency in `segmentProduct()` - it is redundant

**Comment by edwardupton** [comment]
@lyykfi you are too fast! I was committing fixes for your issues as you were writing

**Comment by lyykfi** [comment]
@edwardupton that is my style :-) Thank you for fixing. Looks good!

**Pull request merged**